leaflet-treering.js

```diff
@@ -221,43 +228,47 @@ function MeasurementData (dataObject) {
   * Add a new point into the measurement data
   * @function newPoint
   */
-  MeasurementData.prototype.newPoint = function(start, latLng, hasLatewood) {
+  MeasurementData.prototype.newPoint = function(start, latLng, subAnnual, forwardDirection) {
```

Comment:
you shouldn't be passing subAnnual and forwarddireciton into all these methods - they're available in other ways.